ideas/290-hardware-wallet-light

```diff
@@ -0,0 +1,123 @@
+---
+id: 290-hardware-wallet-light
+title: Hardware Wallet Light
+status: Active
+created: 2018-08-24
+category: core
+lead-contributor: bitgamma
+contributors:
+    bitgamma
+    gravityblast
+    goranjovic
+    dmitryn
+    patrick
+    denis-sharpyn
+    guylouis
+    
```

Comment:
I would suggest adding @obi2020 as a content strategist. We need to explain a lot to users.
Also, we have a marketing website about a wallet. It has to be updated as well if anything has changed https://hardwallet.status.im/